src/D2L.CodeStyle.Analyzers/ApiUsage/Events/EventPublisherEVentTypesAnalyzer.cs

```diff
@@ -123,8 +117,8 @@ string publisherTypeName
 				return Enumerable.Empty<ISymbol>();
 			}
 

```

Comment:
All the methods were accounted for in each case, so simplifying and grabbing all methods